e2e/tests/api-driven/src/permissions/queries/index.ts

```diff
@@ -0,0 +1,7 @@
+import { INSERT_FLOW_QUERY } from "./flows";
+
+export const queries = {
```

Comment:
This PR is just setup. I envision the next PR will be just a bunch of queries (and maybe a few additional setup/cleanup steps)